Scripts/assignment/emme_assignment.py

```diff
@@ -154,6 +154,7 @@ def aggregate_results(self, resultdata):
         # Aggregate and print vehicle kms and link lengths
         vdfs = {param.roadclasses[linktype].volume_delay_func
             for linktype in param.roadclasses}
+        vdfs.add(0)
```

Comment:
This looks a bit funny to me. Is there some other way to add "links without car traffic (= without volume delay function)" than manually?